src/elements/content-sidebar/__tests__/DetailsSidebar.test.js

```diff
@@ -2,13 +2,18 @@ import { shallow } from 'enzyme';
 import * as React from 'react';
 import messages from '../../common/messages';
 import { getBadItemError } from '../../../utils/error';
-import { SIDEBAR_FIELDS_TO_FETCH } from '../../../utils/fields';
+import { SIDEBAR_FIELDS_TO_FETCH, SIDEBAR_FIELDS_TO_FETCH_ARCHIVE } from '../../../utils/fields';
+import { isFeatureEnabled } from '../../common/feature-checking';
 import { DetailsSidebarComponent as DetailsSidebar } from '../DetailsSidebar';
 
 jest.mock('../SidebarFileProperties', () => 'SidebarFileProperties');
 jest.mock('../SidebarAccessStats', () => 'SidebarAccessStats');
 jest.mock('../SidebarClassification', () => 'SidebarClassification');
 jest.mock('../SidebarContentInsights', () => 'SidebarContentInsights');
+jest.mock('../../common/feature-checking', () => ({
+    ...jest.requireActual('../../common/feature-checking'),
```

Comment:
Do we need the rest of the module or can we just mock the whole thing?

```
jest.mock('../../common/feature-checking');
```

Reply:
Fixed here and in `ContentSidebar.test.js`